src/Promitor.Agents.ResourceDiscovery/Dockerfile.linux

```diff
@@ -13,13 +13,61 @@ COPY Promitor.Integrations.LogAnalytics/* Promitor.Integrations.LogAnalytics/
 COPY Promitor.Integrations.Sinks.Prometheus/* Promitor.Integrations.Sinks.Prometheus/
 RUN dotnet publish Promitor.Agents.ResourceDiscovery/Promitor.Agents.ResourceDiscovery.csproj --configuration release --output /app /p:Version=$VERSION
 
-FROM mcr.microsoft.com/dotnet/aspnet:7.0.7-alpine3.17 AS runtime
+FROM mcr.microsoft.com/dotnet/aspnet:7.0.7-cbl-mariner2.0-distroless AS runtime-base
```

Comment:
Any reason why we define this here already while we only use it only later on?

Reply:
It's for copying the root folder to the installer stage, where we install ICU, clean things up, and copy it back to the runtime stage:

```
...
FROM mcr.microsoft.com/cbl-mariner/base/core:2.0 AS installer
...
COPY --from=runtime-base / /staging1
COPY --from=runtime-base / /staging2

...
FROM runtime-base AS runtime
COPY --from=installer /staging2/ /
...
```

Reply:
Makes sense, I looked over that - thanks!